src/form/FormAPI/CustomForm.php

```
@@ -0,0 +1,173 @@
+        $this->addContent(["type" => "input", "text" => $text, "placeholder" => $placeholder, "default" => $default]);
+        $this->labelMap[] = $label ?? count($this->labelMap);
+        $this->validationMethods[] = static fn($v) => is_string($v);
+        return $this;
+    }
+
+    /**
+     * @param array<string, mixed> $content
+     * @return $this
+     */
+    private function addContent(array $content) : self {
+        $this->data["content"][] = $content;
```

PHPStan (2.1.15)

Cannot access an offset on mixed.


(offsetAccess.nonOffsetAccessible)